staff/francisco-fernandez/app-template/about.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About</title>
+</head>
+<body>
+        <h1>About Us</h1>
```

Comment:
For better semantics, include ```<h1>```in the tag ```<header>```inside ```<body>```

Reply:
done